packages/i18n/src/index.js

```diff
@@ -4,11 +4,6 @@
 import Jed from 'jed';
 import memoize from 'memize';
 
-/**
- * WordPress dependencies
- */
-import deprecated from '@wordpress/deprecated';
```

Comment:
This package no longer depends on deprecated, so it should be removed from the `package.json` file and from `client-assets.php`.